lib/engine/game/g_1848/game.rb

```diff
@@ -270,8 +292,8 @@ class Game < Game::Base
           {
             sym: 'P1',
             name: "Melbourne & Hobson's Bay Railway Company",
-            value: 40,
-            discount: 10,
+            value: 30,
+            discount: 0,
```

Comment:
Nitpick: If the discount is just 0 I believe you can simply omit the discount entirely